src/cc65/preproc.c

```diff
@@ -3156,7 +3179,8 @@ static int ParseDirectives (unsigned ModeFlags)
                     case PPD_PRAGMA:
                         if (!PPSkip) {
                             if ((ModeFlags & MSM_IN_ARG_LIST) == 0) {
-                                DoPragma ();
+                                DoPragma();
```

Comment:
Why do you remove the space before the parenthesis? This is not the coding style of cc65.